crates/biome_js_analyze/tests/specs/nursery/useExplicitTestAssertions/valid.js

```diff
@@ -0,0 +1,52 @@
+/* should not generate diagnostics */
+it.todo('will test something eventually');
+
+test.todo('will test something eventually');
+
+['x']();
+
+it('should pass', () => expect(true).toBeDefined());
+
+test('should pass', () => expect(true).toBeDefined());
```

Comment:
These test cases were pulled from the original `expect-expect` ESLint rule, so these are all correct. Now to just get them passing

Reply:
There’s [one test case from the original](https://github.com/jest-community/eslint-plugin-jest/blob/main/src/rules/__tests__/expect-expect.test.ts#L21) that may be beyond my abilities to get passing:

```js
it("should pass", myTest);
function myTest() {
  expect(true).toBeDefined();
}
```

The current logic just does a shallow scan for `expect()` at the top level, but doesn’t trace references within the same file. Would love help with this, if it’s trivial. Or we could punt on it for now and just wait till someone reports it.

Reply:
I think we can leave it for future oppotunities.